packages/test/test-end-to-end-tests/src/test/mapEndToEndTests.spec.ts

```diff
@@ -374,6 +381,74 @@ describeNoCompat("SharedMap orderSequentially", (getTestObjectProvider) => {
             clearEventCount++;
         });
     });
+    describe("Concurrent op processing", () => {
+        let container2: Container;
+        let dataObject2: ITestFluidObject;
+        let sharedMap2: SharedMap;
+
+        beforeEach(async () => {
+            await reset();
+            provider = getTestObjectProvider();
+        });
+
+        afterEach(async () => reset());
```

Comment:
Redundant with the call to `reset` in `beforeEach` (I'd remove that and leave this)